packages/base/src/dialogs/ProcessingFormDialog.tsx

```diff
@@ -157,9 +170,14 @@ export class ProcessingFormDialog extends Dialog<IDict> {
     formErrorSignal.connect((_, extraErrors) => {
       const invalid = extraErrors || !!this.node.querySelector(':invalid');
       if (invalid) {
-        this.node
-          .getElementsByClassName('jp-mod-accept')[0]
-          .setAttribute('disabled', '');
+        const okBtn = this.node.getElementsByClassName(
+          'jp-mod-accept',
+        )[0] as HTMLButtonElement;
+        if (invalid) {
+          okBtn.setAttribute('disabled', '');
+        } else {
+          okBtn.removeAttribute('disabled');
+        }
```

Comment:
This can go back to the way it was. This change checks `if (invalid)` when we already know `invalid` is truthy:

```typescript
if (invalid) {
  // ...
  if (invalid) {
    // We already know this is true
  } else {
    // We handle the else case just below
  }
} else {
  //...
}
```